src/backend/gl/src/device.rs

```diff
@@ -381,19 +368,13 @@ impl d::Device<B> for Device {
         unimplemented!()
     }
 
-    fn read_mapping_raw(&mut self, _: &n::Buffer, _: Range<u64>)
-        -> Result<(*const u8, Mapping), mapping::Error>
-    {
-        unimplemented!()
-    }
-
-    fn write_mapping_raw(&mut self, _: &n::Buffer, _: Range<u64>)
-        -> Result<(*mut u8, Mapping), mapping::Error>
+    fn acquire_mapping_raw(&mut self, buf: &n::Buffer, read: Option<Range<u64>>)
+        -> Result<*mut u8, mapping::Error>
     {
         unimplemented!()
     }
 
-    fn unmap_mapping_raw(&mut self, _: Mapping) {
+    fn release_mapping_raw(&mut self, buf: &n::Buffer, wrote: Option<Range<u64>>) {
```

Comment:
I'd be curious if GL fits the current design. Having some rough implementation here would be great as a proof of concept.